### ISSUE TITLE
HTTP 409 error: Unable to delete record:

### ISSUE THREAD

**Issue by FlyingDiver** [FlyingDiver/Indigo-Twilio#7]
```
   Trigger                         Twilio Webhook
   Twilio Debug                    checkMessages: Message from +14083688096, to: +12392179488, direction: inbound, date_sent: 'None'
   Twilio Error                    checkMessages: twilioClient.messages.delete() error: HTTP 409 error: Unable to delete record: Cannot delete message because delivery has not been completed.
Traceback (most recent call last):
  File "plugin.py", line 433, in checkMessages
    self.twilioClient.messages(message.sid).delete()
  File "/Library/Application Support/Perceptive Automation/Indigo 7.5/Plugins/Twilio.indigoPlugin/Contents/Server Plugin/twilio/rest/api/v2010/account/message/__init__.py", line 326, in delete
    return self._version.delete('delete', self._uri)
  File "/Library/Application Support/Perceptive Automation/Indigo 7.5/Plugins/Twilio.indigoPlugin/Contents/Server Plugin/twilio/base/version.py", line 124, in delete
    raise self.exception(method, uri, response, 'Unable to delete record')
TwilioRestException: HTTP 409 error: Unable to delete record: Cannot delete message because delivery has not been completed.
   Twilio Debug                    checkMessages: Done

```

**Comment by FlyingDiver** [comment]
So I think I fixed the 409 error exception, but not the underlying cause. It looks like the Twilio servers are firing the webhook before all the messages are fully processed. I get the new message "object" back from the server, but it's not complete. No date_sent field, for instance. And attempting to process and delete it at that time fails. For now, the plugin won't even attempt to process it, but since the webhook has already fired, it will not be re-fetched and processed until another webhook fires, or it polls (interval set in plugin config).

I have a ticket open with Twilio to see why the webhook is firing early.

**Comment by FlyingDiver** [comment]
Fixed in 7.2.2.
